SimG4CMS/Calo/test/python/minbias_cfg.py

```diff
@@ -6,7 +6,7 @@
 process.load("IOMC.EventVertexGenerators.VtxSmearedGauss_cfi")
 process.load("Geometry.CMSCommonData.cmsIdealGeometryXML_cfi")
 process.load("Geometry.TrackerNumberingBuilder.trackerNumberingGeometry_cfi")
-process.load("Configuration.StandardSequences.MagneticField_cff")
+process.load("Configuration.StandardSequences.MagneticField_38T_cff")
```

Comment:
I understood that I needed some global tag for the B-field. It was giving error otherwise. While I had to change that I  found 3.8T is a better option. I am not sure if this change was absolutely necessary for the run - but addition of data base and global tag was needed
________________________________
From: David Lange [notifications@github.com]
Sent: 25 June 2015 17:21
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2 hcx14 bring in simualtion level of geometry changes to 7_X version (#9833)


In SimG4CMS/Calo/test/python/minbias_cfg.py<https://github.com/cms-sw/cmssw/pull/9833#discussion_r33266857>:

> @@ -6,7 +6,7 @@
>  process.load("IOMC.EventVertexGenerators.VtxSmearedGauss_cfi")
>  process.load("Geometry.CMSCommonData.cmsIdealGeometryXML_cfi")
>  process.load("Geometry.TrackerNumberingBuilder.trackerNumberingGeometry_cfi")
> -process.load("Configuration.StandardSequences.MagneticField_cff")
> +process.load("Configuration.StandardSequences.MagneticField_38T_cff")


@bsunanda<https://github.com/bsunanda> - why are you making this change?

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/9833/files#r33266857>.